Yoast/wordpress-seo
# Bumped lodash to version 4.17.11

## Summary

This PR can be summarized in the following changelog entry:

* [not-userfacing] Updated lodash to version 4.17.11.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* No specific tests that can be done. Click through the application and make sure nothing breaks.

## UI changes

## Quality assurance


Fixes #
